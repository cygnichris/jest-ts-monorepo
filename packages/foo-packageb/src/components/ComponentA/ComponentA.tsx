import * as React from 'react';

import { ComponentB } from '@bsrc/components'
export const ComponentA = () => <div><ComponentB /></div>
