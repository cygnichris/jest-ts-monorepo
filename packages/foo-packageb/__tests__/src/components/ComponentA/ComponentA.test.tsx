import * as React from 'react';
import { render } from '@testing-library/react';
import { ComponentA } from '@bsrc/components/ComponentA/ComponentA';


describe('<ComponentA/> Snapshot', () => {
  it('renders', () => {
    const { asFragment } = render(
      <ComponentA />
    );

    expect(asFragment()).toMatchSnapshot();
  })
})


