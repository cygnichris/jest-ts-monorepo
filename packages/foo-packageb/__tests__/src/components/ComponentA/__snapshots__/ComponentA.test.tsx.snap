// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ComponentA/> Snapshot renders 1`] = `
<DocumentFragment>
  <div>
    <div>
      <div>
        Token One: One
      </div>
      <div>
        Hello World
      </div>
    </div>
  </div>
</DocumentFragment>
`;
